modules/history.js

```diff
@@ -0,0 +1,166 @@
+function History() {
+  // Initialize history to either the stored values or defaults
+  if (!this.loadHistory()) {
+    this.set(this.getDefault());
+  }
+}
+
+History.prototype.getDefault = function() {
+  return {
+    photos: {
+      title: 'Photos',
+      lastBackupDate: '2015-06-20T19:00-0700',
```

Comment:
I know this is kinda late but I think last backup date default value should be the js value undefined. 
Otherwise we are providing defaults that present incorrect data.